recipe/meta.yaml

```diff
@@ -35,7 +35,6 @@ outputs:
         - liblapack
         - liblapacke
         - log4cxx
-        - openblas
```

Comment:
@erykoff Is the openblas removal deliberate?

Reply:
Good question.  This is a part of conda-forge I don't fully understand.  @beckermr ?

Reply:
It needs to be in there to get the pinning right.

Reply:
I see that openblas was in the run section below, just got dropped from the host section.